ccpp/physics_namelists/input_RRFS_v1nssl.nml

```diff
@@ -0,0 +1,95 @@
+&gfs_physics_nml
+    bl_mynn_edmf = 1
+    bl_mynn_edmf_mom = 1
+    bl_mynn_tkeadvect = .true.
+    cal_pre = .false.
+    cdmbgwd = 3.5,0.25
+    cnvcld = .false.
+    cnvgwd = .false.
+    cplflx = .false.
+    debug = .true.
+    do_deep = .false.
+    do_mynnedmf = .true.
+    do_mynnsfclay = .false.
+    do_shum = .false.
+    do_skeb = .false.
+    do_sppt = .false.
+    dspheat = .true.
+    effr_in = .true.
+    fhcyc = 0.0
+    fhlwr = 1200.0
+    fhswr = 1200.0
+    fhzero = 0.16667
+    h2o_phys = .true.
+    hybedmf = .false.
+    iaer = 111
+    ialb = 1
+    iau_delthrs = 6
+    iau_inc_files = ''
+    iaufhrs = 30
+    icloud_bl = 1
+    ico2 = 2
+    iems = 1
+    imfdeepcnv = -1
+    imfshalcnv = -1
+    imp_physics = 17
+    nssl_ccn_on = .false.
```

Comment:
Is this namelist supposed to have nssl_ccn_on and nssl_hail_on = F? The other one (input_RRFS_v1_nssl_nohailnoccn.nml) already has these as F.

Reply:
Good catch, I probably forgot I had changed it during testing. Since the default for the suite is

"SCM_RRFS_v1nssl":"input_RRFS_v1nssl_nohailnoccn.nml"

Then I could either remove input_RRFS_v1nssl.nml or change it to turn on both hail and CCN as an example. Although I suppose the point of SCM is not to get the 'best' result but rather to check that the scheme is working.  Generally, I'd like to set things up for the intended use/needs of SCM. Hail was turned off here because the scheme probably would not produce any, and it's not an output variable anyway.